domains/manav.json

```diff
@@ -0,0 +1,11 @@
+{ 
+   "owner": { 
+     "username": "Manav011", 
+     "email": "manavbafna87+Github@gmail.com" 
+   }, 
+   "record": { 
+     "A": ["109.71.252.37"], 
```

Comment:
I cannot reach your site
"109.71.252.37 took too long to respond."

Reply:
Resolved, can you please check once